src/main/java/org/commonwl/view/workflow/WorkflowController.java

```diff
@@ -214,7 +214,7 @@ public ModelAndView getWorkflow(
    * @param branch The branch of the repository
    * @return The workflow view with the workflow as a model
    */
-  @GetMapping(value = "/workflows/**/*.git/{branch}/**")
+  @GetMapping(value = "/workflows/*/*.git/{branch}/**")
```

Comment:
This needs to be well tested.

Now Spring Web doesn't allow any content after the `/**`. I found an issue or blog that suggested to test with `/*/bla` instead. Trying to get the tests and build to pass in order to verify it in the browser.